examples/column_view_datagrid/grid_cell/mod.rs

```diff
@@ -0,0 +1,28 @@
+mod imp;
+use gtk::glib;
+use gtk::subclass::prelude::*;
+
+glib::wrapper! {
+    pub struct GridCell(ObjectSubclass<imp::GridCell>)
+        @extends gtk::Widget, gtk::Box;
+}
+
+impl Default for GridCell {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+pub struct Entry {
+    pub name: String,
+}
+
+impl GridCell {
+    pub fn new() -> Self {
+        glib::Object::new(&[]).expect("Failed to create GridCell")
+    }
+
+    pub fn set_app_info(&self, app_info: &Entry) {
```

Comment:
`app_info` seems a really specific name, but actually this example has really generic strings, so I suggest using a more generic name. `entry` seems a fine name and it's also consistent with the struct name.

Reply:
indeed, that is carry-over from the examples/list_view_apps_launcher/ which i used as a base for some of this in the app. renamed to entry and set_entry